recipes/boost/all/conanfile.py

```diff
@@ -192,7 +249,9 @@ def _run_python_script(self, script):
             self.output.info("(failed)")
             return None
         output = output.getvalue().strip()
-        self.output.info(output)
+        # Conan is broken when run_to_output = True
+        if "\n-----------------\n" in output:
```

Comment:
probably, this deserves an issue?

Reply:
It does.
I thought there was already one, but I cannot find it.

Reply:
okay, feel free to create, if it's missing. if we find it later, we always can close duplicate.

Reply:
I found it:
https://github.com/conan-io/conan/issues/6752